devmt-horizontal-simple.svg

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   viewBox="0 0 1333.3333 533.33331"
+   height="533.33331"
+   width="1333.3333"
+   xml:space="preserve"
+   id="svg2"
+   version="1.1"><metadata
+     id="metadata8"><rdf:RDF><cc:Work
+         rdf:about=""><dc:format>image/svg+xml</dc:format><dc:type
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" /></cc:Work></rdf:RDF></metadata><defs
+     id="defs6" /><g
+     transform="matrix(1.3333333,0,0,-1.3333333,0,533.33333)"
+     id="g10"><text
+       id="text14"
+       style="font-variant:normal;font-weight:bold;font-stretch:normal;font-size:151.81129456px;font-family:Aller;-inkscape-font-specification:Aller-Bold;writing-mode:lr-tb;fill:#e64c3c;fill-opacity:1;fill-rule:nonzero;stroke:none"
```

Comment:
Se o usuário não tiver a font no computador instalada, a logo carrega a font padrão do browser... ai fica estranho.

Pra isso, tem que vetorizar (transformar os textos em curvas) a logo antes de virar SVG.. acredito que o Illustrator ou alguma outra ferramenta deva fazer isso sem muitas dificuldades

Reply:
eu fiz por esse site, https://cloudconvert.com/ai-to-svg
não tenho o illustrator, bem q vi que estava meio bugado as imagens